include/moveit_visual_tools/moveit_visual_tools.h

```diff
@@ -323,33 +323,40 @@ class MoveItVisualTools : public rviz_visual_tools::RvizVisualTools
                              const rviz_visual_tools::colors& color = rviz_visual_tools::GREEN);
 
   /**
-   * \brief Create a MoveIt Collision block at the given pose
+   * \brief Create/Publish a MoveIt Collision block at the given pose
    * \param pose - location of center of block
    * \param name - semantic name of MoveIt collision object
    * \param size - height=width=depth=size
    * \param color to display the collision object with
    * \return true on sucess
    **/
+  moveit_msgs::CollisionObject createCollisionBlock(const geometry_msgs::Pose& block_pose, const std::string& name,
+                                                    double block_size);
+
   bool publishCollisionBlock(const geometry_msgs::Pose& block_pose, const std::string& block_name = "block",
                              double block_size = 0.1,
                              const rviz_visual_tools::colors& color = rviz_visual_tools::GREEN);
 
   /**
-   * \brief Create a MoveIt collision rectangular cuboid at the given pose
+   * \brief Create/Publish a MoveIt collision rectangular cuboid at the given pose
    * \param point1 - top left of rectangle
    * \param point2 - bottom right of rectangle
    * \param name - semantic name of MoveIt collision object
    * \param color to display the collision object with
    * \return true on sucess
    **/
+  moveit_msgs::CollisionObject createCollisionObject(const Eigen::Vector3d& point1, const Eigen::Vector3d& point2,
```

Comment:
this should be ``createCollisionCuboid`` right? Even better: ``createCollisionCuboidMsg``

Reply:
done